### PR TITLE
Community Standards

### PR DESCRIPTION
**Pull request by jmesnil**
* add Code of conduct, contributing and security document
* add Getting Help section to the README

Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Contributing guide
+
+**Want to contribute? Great!** 
+We try to make it easy, and all contributions, even the smaller ones, are more than welcome.
+This includes bug reports, fixes, documentation, examples... 
+But first, read this page (including the small print at the end).
+
+## Legal
+
+All original contributions to Jandex licensed under the
```
Comment (Ladicek) [inline CONTRIBUTING.md:10]:
```suggestion
All original contributions to Jandex are licensed under the
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+[ASL - Apache License](https://www.apache.org/licenses/LICENSE-2.0),
+version 2.0 or later, or, if another license is specified as governing the file or directory being
+modified, such other license.
```
Comment (Ladicek) [inline CONTRIBUTING.md:13]: Any reason why you dropped the DCO paragraph here? I don't really understand the difference, just asking :-)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Build
+
+* Clone the repository: `git clone https://github.com/smallrye/jandex.git`
+* Navigate to the directory: `cd jandex`
+* Invoke `mvn clean install` from the root directory
+
+```bash
+git clone https://github.com/smallrye/jandex.git
+cd smallrye-mutiny
```
Comment (Ladicek) [inline CONTRIBUTING.md:62]:
```suggestion
cd jandex
```

**Comment by Ladicek** [comment]
Otherwise looks good. Thanks!

File: CONTRIBUTING.md (hunk 1 above)
Comment (jmesnil) [reply CONTRIBUTING.md]: fixed, thanks

File: CONTRIBUTING.md (hunk 2 above)
Comment (jmesnil) [reply CONTRIBUTING.md:13]: There is no dco.txt file in Jandex so I was not sure if it applied.
If you want I can add https://github.com/smallrye/smallrye-mutiny/blob/main/dco.txt to this PR and put back the DCO paragraph

File: CONTRIBUTING.md (hunk 3 above)
Comment (jmesnil) [reply CONTRIBUTING.md]: I can't make a copy/paste without errors... fixed, thanks

File: CONTRIBUTING.md (hunk 2 above)
Comment (Ladicek) [reply CONTRIBUTING.md:13]: Please do, thanks!

Comment (jmesnil) [reply CONTRIBUTING.md:13]: done

**Review by Ladicek — APPROVED** [review]

**Pull request merged**